### ISSUE TITLE
Constant strings as an argument

### ISSUE THREAD

**Issue by Koudysmen** [StubbleOrg/Stubble.Helpers#27]
Hi guys,

I have a problem with my custom Helper if i using more than one constant string as an argument in a row then my helper is not invoked. 

Helper register
```
helpers.Register("ReplaceString", (HelperContext context, string searchString, string oldString, string newString) => searchString?.Replace(oldString, newString, StringComparison.InvariantCulture));
```

Render template
```
renderer.Render("Name: {{ReplaceString Name 'XXX' ''}}", new
            {
                Name = "JohnXXX"
            });
```
Basically I want to replace XXX with empty string. But the result is just `Name: ` So there are something wrong.

But when I changed the order of the arguments in the template then my custom helper working correctly.

Helper register
```
helpers.Register("ReplaceString", (HelperContext context, string oldString, string searchString, string newString) => searchString?.Replace(oldString, newString, StringComparison.InvariantCulture));
```

Render template
```
renderer.Render("Name: {{ReplaceString  'XXX' Name ''}}", new
            {
                Name = "JohnXXX"
            });
```
Then result is correct:  `Name: John`

In my assumption stubble helper does not support more than one constant string as an argument in a row currently. It will be great when stubble helper will be support it because this functionality would be useful for us.

Thanks, Michal

**Comment by Romanx** [comment]
Hi,

Thanks for the comprehensive report, this looks like a genuine bug since this should be an accepted case. I'll take a look as soon as I can,

Thanks for using Stubble 👍

**Comment by Romanx** [comment]
Hey,

Thanks for reporting this. It was actually a one line fix in the end but we added tests around this area to make sure we don't regress. The fix is now on nuget in release 1.1.3

Thanks for using Stubble and let us know if you have any issues!

**Comment by Koudysmen** [comment]
Hi, 
Great, Thanks a lot.
